### PR TITLE
Release notes: edits on "Py3 Support" and "New features" sections

### PR DESCRIPTION
**Pull request by stummjr**

### CODE REVIEW

File: docs/news.rst [hunk 1]
```diff
@@ -48,50 +45,27 @@ New Features and Enhancements
...
+    + `optional_features` has been removed #1699
 - Enable robots.txt handling by default for new projects. #1724 (https://github.com/scrapy/scrapy/commit/0d368c5d6fd468aed301ed5967f8bfe9d5e86101)
-    WARNING: backwards incompatible
-
-- Disable CloseSpider extension if no CLOSPIDER_* setting set #1723 (https://github.com/scrapy/scrapy/commit/2246280bb6f71d7d52e24aca5b4ce955b3aa1363)
-- Disable SpiderState extension if no JOBDIR set #1725
-
-- Add Code of Conduct Version 1.3.0 from http://contributor-covenant.org/ #1681
+    **Warning: backwards incompatible**
+- CloseSpider extension is disabled if no `CLOSEPIDER_*` setting is defined #1723 (https://github.com/scrapy/scrapy/commit/2246280bb6f71d7d52e24aca5b4ce955b3aa1363)
```
Comment (eliasdorneles) [inline docs/news.rst]: typo fixing a typo? :)
`CLOSEPIDER_*` > `CLOSESPIDER_*`

Comment (stummjr) [inline docs/news.rst]: thanks for the catch! :D

File: docs/news.rst [hunk 2]
```diff
@@ -48,50 +45,27 @@ New Features and Enhancements
...
-    Adds a nested_xpath()/nested_css() methods to ItemLoader. (TODO: add links to docs)
-
-- add_scheme_if_missing for `scrapy shell` command #1498 (https://github.com/scrapy/scrapy/commit/fe15f93e533be36e81e0385691fe5571c88b0b31)
-    Fixes: #1487
-    Warning: backward incompatible
-
-    + see: https://github.com/scrapy/scrapy/issues/1550, https://github.com/scrapy/scrapy/pull/1710
+- `FormRequest.from_response` now allows to define through CSS selectors which form from the response should be used. It previously supported only XPath. #1382 (https://github.com/scrapy/scrapy/commit/a6e5c848feb672c117f3380976077b6d0f42e3a6)
+- Scrapy doesn't retry requests that got `400 Bad Request` reponse anymore. #1289 (https://github.com/scrapy/scrapy/milestones/Scrapy%201.1) **Warning: backwards incompatible!**
+    + Fix docs for #1289 in #1530 (https://github.com/scrapy/scrapy/commit/451318ef7a4e8ee7837b83e73b158da98f579980)
```
Comment (stummjr) [inline docs/news.rst]: I am not sure if "Fix docs" for a feature is relevant, what do you think?

Comment (eliasdorneles) [inline docs/news.rst]: yeah, I'd remove it from the notes.

File: docs/news.rst [hunk 3]
```diff
@@ -9,34 +9,31 @@ Release notes
 Python 3 Support (basic)
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-We have been hard at work to make Scrapy work on Python 3. Some features
-are still missing (and may never be ported to Python 3, see below),
-but you can now run spiders on Python 3.3, 3.4 and 3.5.
+We have been hard at work to make Scrapy run on Python 3. As a result, now you can run spiders on Python 3.3, 3.4 and 3.5, although some features are still missing (some of them may never be ported to Python 3).
```
Comment (kmike) [inline docs/news.rst]: I'm in a 'fit text to 70-80 columns manually, don't rely on auto-wrap features in editors' camp :)

Comment (stummjr) [inline docs/news.rst]: :+1:

**Comment by redapple** [comment]
Thanks @stummjr

**Pull request merged**